src/Wizard.js

```diff
@@ -105,6 +105,11 @@ export default class Wizard extends Webform {
       showCancel: !this.options.readOnly
     });
 
+    if (!this.isSecondInit) {
+      this.isClickableDefined = _.clone(this.options?.breadcrumbSettings?.hasOwnProperty('clickable'));
```

Comment:
Why do you need to _.clone this?...  It is set to a boolean so therefore is not a reference.